app/views/categories/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_with model: @category, url: categories_path do |f| %>
+  <section class="h-full bg-white min-h-screen">
+    <div class=" py-6  h-full ml-4 mr-4">
+      <div class="mb-4">
+          <%= f.text_field :name, autofocus: true, autocomplete: "email", placeholder: 'Name',class:"form-control block w-full px-3 py-1.5 text-base font-normal text-gray-700 bg-white bg-clip-padding border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+          %>
+        </div>
+        <div class="mb-4">
+          <%= f.text_field :icon, autofocus: true, autocomplete: "email", placeholder: 'Icon url',class:"form-control block w-full px-3 py-1.5 text-base font-normal text-gray-700 bg-white bg-clip-padding border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+          %>
+        </div>
+        <div class="mb-4">
+          <%= f.submit "Save", class:"form-control block w-full px-3 py-1.5 text-lg font-normal text-white bg-main-second border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+          %>
+        </div>
+    </div>
+   </section>
+<% end %>
```

Comment:
###  "Add a new category" page.

- [ ] Great job on the form to create a new category. 👏🏻  However you should have a back button so that when the user clicks on the "Back" button (<), the user navigates to the home page,. Let's add this 👍🏻